ui/src/controller/trace_controller.ts

```diff
@@ -526,7 +526,6 @@ export class TraceController extends Controller<States> {
       publishFtraceCounters(counters);
     }
 
-    globals.dispatch(Actions.sortThreadTracks({}));
```

Comment:
IIRC, originally in Perfetto, only processes were grouped, so this only sorted tracks in those process groups. But now we group lots else. Are we losing the sorting of more than just threads by this change? The name of the function is `sortThreadTracks` but it explicitly handles non-thread track sorting, also — via the `PrimaryTrackSortKey` — for tracks that don't have a `ThreadTrackSortKey`.

Reply:
The ask was to remove alphabetical sorting, noticed in the GPU counters section.  I noticed that this code ends up sorting all tracks alphabetically after they have been added.  I don't think that this action should effect non-thread tracks in any way, so I added a check where if either track in the sort does not have a ThreadSortKey it is simply skipped.  This should preserve the ThreadTrack sort while not effecting other tracks like GPU Counters.